src/components/Label/tests/Label.test.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import {mount} from 'enzyme';
+import Label from '../Label';
+
+describe('Label Suit', () => {
+
+    it('Verify Label is not Hidden', () => {
+          const labelWrapper = mount(<Label id="lblid" hidden={true} />);
+          expect(labelWrapper.prop('hidden')).toBe(true);
+    });
+
+    it('Verify Label is  Hidden', () => {
+          const labelWrapper = mount(<Label id="lblid" hidden={false} />);
+          expect(labelWrapper.prop('hidden')).toBe(false);
+    });
+
+    it('Verify type of hidden', () => {
+      const labelWrapper = mount(<Label id="lblid" hidden={false}/>).prop('hidden');
+      expect(typeof (labelWrapper)).toBe('boolean');
```

Comment:
@harsimranb @mkulinski I am not sure about this. Is this a good practice to test type of property in react? Do we need to test type of property? Lets discuss this.